frontend/integration-tests/tests/monitoring.scenario.ts

```diff
@@ -215,10 +215,205 @@ describe('Monitoring: YAML', () => {
+    );
+    expect(monitoringView.firstListLinkById('dropdown-button').getText()).toEqual('PagerDuty');
+    expect(monitoringView.firstListLinkById('pagerduty-key-label').getText()).toEqual(
+      'Service Key',
+    );
+    expect(monitoringView.firstListLinkById('integration-key').getAttribute('value')).toEqual(
+      '<integration_key>',
+    );
+    expect(monitoringView.firstListLinkById('label-name-0').getAttribute('value')).toEqual(
+      'severity',
+    );
+    expect(monitoringView.firstListLinkById('label-value-0').getAttribute('value')).toEqual(
+      'warning',
+    );
+
+    // Edit Values
+
+    await monitoringView.firstListLinkById('receiver-name').clear();
+    await monitoringView.firstListLinkById('receiver-name').sendKeys('MyEditedReceiver');
+    await monitoringView
+      .firstListLinkById('label-name-0')
+      .clear()
+      .then(() => monitoringView.firstListLinkById('label-name-0').sendKeys('cluster'));
+    await monitoringView
+      .firstListLinkById('label-value-0')
+      .clear()
+      .then(() => monitoringView.firstListLinkById('label-value-0').sendKeys('MyCluster'));
+
+    await monitoringView.saveButton.click();
+    await crudView.isLoaded();
+
+    monitoringView.getFirstRowAsText().then((text) => {
+      expect(text).toEqual('MyEditedReceiver pagerduty cluster = MyCluster');
+    });
+  });
+
+  it('deletes PagerDuty Receiver correctly', async () => {
+    await crudView.isLoaded();
+    expect(crudView.resourceRows.count()).toBe(2);
+
+    await monitoringView.clickFirstRowKebabAction('Delete Receiver');
+    await browser.wait(until.presenceOf(monitoringView.saveButton));
+    await monitoringView.saveButton.click();
+
+    await crudView.isLoaded();
+    expect(crudView.resourceRows.count()).toBe(1);
+  });
+
+  it('prevents deletion of default receiver', async () => {
+    await crudView.isLoaded();
+    await monitoringView.openFirstRowKebabMenu();
+    expect(monitoringView.disabledKebabMenuItem.getText()).toEqual('Delete Receiver');
+  });
+
+  it('prevents deletion and form edit of a receiver with sub-route', async () => {
+    // add receiver with sub-route
+    const yaml = `route:
+  routes:
+    - match:
+      service: database
+      receiver: team-DB-pager
+      routes:
+        - match:
+          owner: team-X
+          receiver: team-X-pager
+receivers:
+- name: 'team-X-pager'
+- name: 'team-DB-pager'`;
+
+    await crudView.isLoaded();
+    await horizontalnavView.clickHorizontalTab('YAML');
+    await crudView.isLoaded();
+    await monitoringView.yamlTextArea.clear();
+    await monitoringView.yamlTextArea.sendKeys(yaml);
+    await monitoringView.saveButton.click();
+    await crudView.isLoaded();
+    expect(monitoringView.successAlert.isPresent()).toBe(true);
+
+    await horizontalnavView.clickHorizontalTab('Overview');
+    await monitoringView.openFirstRowKebabMenu();
+    expect(monitoringView.disabledKebabMenuItem.getText()).toEqual('Delete Receiver');
```

Comment:
It seems like it would be better to get the delete receiver menu item and check that it's disabled (instead of assuming the first disabled item is delete receiver)

Reply:
When I add `disabled` attribute to the `<button />` the disabled tooltip doesn't render.  However, I have updated the selector to be more specific.